packages/client/src/components/SafeSettings/components/Assets/RemoveVault.js

```diff
@@ -0,0 +1,31 @@
+import { Close } from "components/Svg";
```

Comment:
Since removeVault and removeCollection are almost identical except for the message, anyway we can combine the two?

Reply:
I was also thinking about this too, but I left them separate in case in the future someone wants to differentiate them more than just the header and message.

Reply:
I see, that makes sense, but I would still say DRY principle takes the precedence in this instance. Even when we get to the situation of different content, we should be looking at passing and rendering different properties rather than create two files.